da/test_kzg.py

```diff
@@ -0,0 +1,61 @@
+from itertools import chain, batched
+from random import randrange
+from unittest import TestCase
+
+from eth2spec.deneb.mainnet import BLS_MODULUS, bytes_to_bls_field
+
+from da.kzg_rs import kzg
+from da.kzg_rs.common import BYTES_PER_FIELD_ELEMENT, GLOBAL_PARAMETERS, ROOTS_OF_UNITY, GLOBAL_PARAMETERS_G2
+from da.kzg_rs.trusted_setup import verify_setup
+
+
+class TestKZG(TestCase):
+
+    @staticmethod
+    def rand_bytes(size=1024):
```

Comment:
I'm late to review this PR, but have a quick question. It seems like this parameter `size` should be `n_chunks` if I understand correctly?

Reply:
yes it is